dspace-uclouvain/src/main/java/org/dspace/uclouvain/authority/UCLouvainAuthorAuthority.java

```diff
@@ -93,23 +93,13 @@ public DialPerson[] getStudentList(String query) {
     private Map<String, String> generateExtras(DialPerson person){
         Map<String, String> extras = new HashMap<>();
 
-        String entity = person.getEntity();
-        Orcid orcid = new Orcid(person.orcidId);
         String employeeId = person.getEmployeeId();
         String institution = person.getInstitution();
 
         // String with "data-" will be used to fill other form fields 
-        extras.put("institution-affiliation-name", institution);
         extras.put("data-authors_institution_code", institution);
         extras.put("data-authors_email", person.getEmail());
 
-        if(!entity.isEmpty()){
-            extras.put("institution-entity-name", entity);
-        }
-        if(!orcid.getOrcid().isEmpty()){
-            extras.put("person_identifier_orcid", orcid.getID());
-        }
```

Comment:
ORCID isn't yet used for master thesis project but will be used with "pr" project.
Maybe a good idea to return the orcid URI into a "data" tag that could be used later